core/server/master/src/test/java/alluxio/master/journal/sink/JournalSinkTest.java

```diff
@@ -220,6 +272,17 @@ private long findFile(String name) throws Exception {
     return INVALID_ID;
   }
 
+  private long findFile(String name, String path) throws Exception {
```

Comment:
these methods with the path is strictly more specific than the existing ones, right? Should we just replace the existing tests to use these more specific methods?

Reply:
I would like to differentiate tests for sink interface and tests for path population so it is easy to pinpoint the issue when some test fails.

Reply:
Using the new method in the existing sink tests means that both sink tests and the new path tests will fail if the path population logic breaks, which makes it hard to pinpoint the root cause.